core/src/com/po/fuck/model/Player.java

```diff
@@ -26,6 +28,9 @@ public class Player extends Entity {
                         new BasicMovement(this, DEFAULT_SPEED),
                         Constants.BOOST_DISTANCE, Constants.BOOST_DURATION, Constants.BOOST_COOLDOWN));
         teamTag = Constants.PLAYER_TEAM_TAG;
+        BasicSpriteInfo info = SpriteInfoManager.getBasicSpriteInfo(this.getClass());
+        geometryData.setHeight(info.getFrameHeight());
+        geometryData.setWidth(info.getFrameWidth());
```

Comment:
I think we need to left only the `heigh`, `width` that will represent info about the sprite (currently they are `frameWidth`, `frameHeight`) in our data. And remove info about total sprite size, because it can be calculated through file info.